starkinfra/issuingwithdrawal/__issuingwithdrawal.py

```diff
@@ -69,10 +69,10 @@ def query(external_ids=None, after=None, before=None, limit=None, tags=None, use
     Receive a generator of IssuingWithdrawal objects previously created in the Stark Infra API
     ## Parameters (optional):
     - limit [integer, default None]: maximum number of objects to be retrieved. Unlimited if None. ex: 35
-    - external_ids [list of strings, default []]: external IDs. ex: ["5656565656565656", "4545454545454545"]
     - after [datetime.date or string, default None] date filter for objects created only after specified date. ex: datetime.date(2020, 3, 10)
     - before [datetime.date or string, default None] date filter for objects created only before specified date. ex: datetime.date(2020, 3, 10)
     - tags [list of strings, default None]: tags to filter retrieved objects. ex: ["tony", "stark"]
+    - external_ids [list of strings, default []]: external IDs. ex: ["5656565656565656", "4545454545454545"]
```

Comment:
Definir como padrão 

`list of strings, default None` para query string
E 
`list of strings, default []` para atributo de objeto